test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +35,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        Object[] empty = new Object[0];
+        assertNoNull();
+        assertNoNull(1, 1, 1);
+
+        assertAnyNull((Object) null);
+        assertAnyNull(null, null);
+        assertAnyNull(1, 1, null);
+        assertAnyNull(1, 1, 1, null);
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
-
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    private void assertAnyNull(Object... objects) { assertTrue(Utils.isAnyNull(objects)); }
```

Comment:
Not permitted by the coding standard. Refer to the coding standard for using K&R style brackets (aka Egyptian style).